.prettierrc.json

```diff
@@ -1 +1 @@
-{ "quoteProps": "preserve" }
+{ "quoteProps": "preserve", "proseWrap": "always" }
```

Comment:
for the future: it would have been better to separate re-wrapping and content changes into separate PRs